docs/en/rules/Azure.RSV.ReplicationAlert.md

```diff
@@ -59,6 +59,8 @@ alert via a Bicep either configure the `sendToOwners` or `CustomerEmailAddress`
 - Set `properties.sendToOwners` to `Send`.
 - Set `properties.customEmailAddresses` to `[ "example@email.com" ]`
 
+with the locale property you can define the locale for the email notification.
```

Comment:
Lets add this to a separate `## NOTES` section so that the required steps are clear.

Add to notes similar to: https://github.com/Azure/PSRule.Rules.Azure/blob/main/docs/en/rules/Azure.Defender.SQL.md#notes

Reply:
added the ## NOTES and moved the text under it